cabal-testsuite/PackageTests/Check/ConfiguredPackage/Paths/AbsolutePath/cabal.out

```diff
@@ -1,6 +1,6 @@
 # cabal check
 These warnings may cause trouble when distributing the package:
-Warning: [no-glob-match] In 'extra-source-files': the pattern '/home/user/file' does not match any files.
+Warning: [glob-missing-dir] In 'extra-source-files': the pattern '/home/user/file' attempts to match files in the directory '/home/user', but there is no directory by that name.
```

Comment:
This will fail if `Check/Warning.hs` is not changed. I am assuming this is still wip.

Reply:
Basically, we now check more consistently in some situations for whether the directory exists, resulting in "directory does not exist" error being triggered before "file does not match".